apps/website/.vuepress/theme/global-components/DocIconsSet.vue

```diff
@@ -33,6 +37,9 @@ export default {
     icons: function () {
       return this.iconSets[this.setName].icons;
     },
+    iconSet: function () {
```

Comment:
where is this being used? Isn't this property calling itself in loop?

Reply:
Yeah, it was left over from some early poc code. I'll remove it in a separate PR since I didn't see this last night.